src/redux/sagas/extensions/extensionEnable.ts

```diff
@@ -71,7 +72,7 @@ function* extensionEnable({
     return yield putError(ActionTypes.EXTENSION_ENABLE_ERROR, error, meta);
   }
 
-  // yield call(refreshExtensions);
+  yield call(refreshEnabledExtension, colonyAddress, extensionId);
```

Comment:
I notice you dropped the `yield call(...)` pattern when doing the refresh in `extensionInstall.ts`. Not certain if there's a preference for one over the other (I'm used to seeing it this way) but should probably keep them consistent.

Reply:
Good spot, fixed!